embedded-kafka/src/main/scala/net/manub/embeddedkafka/ConsumerExtensions.scala

```diff
@@ -3,19 +3,18 @@ package net.manub.embeddedkafka
 import org.apache.kafka.clients.consumer.{ConsumerRecord, KafkaConsumer}
 import org.apache.kafka.common.KafkaException
 
+import scala.jdk.CollectionConverters._
```

Comment:
These classes do not exist in 2.11 or 2.12.
This will break cross-building.

Reply:
Oops, meant to make this a comment.

Reply:
It shouldn't be a problem.  This package is available in the most recent referenced patch releases of 2.11 and 2.12 that this project references.

```
sbt:embedded-kafka-root> ++2.12.10; compile; ++2.11.12; compile;
[info] Setting Scala version to 2.12.10 on 3 projects.
[info] Reapplying settings...
[info] Set current project to embedded-kafka-root (in build file:/home/seglo/source/embedded-kafka/)
[info] Compiling 18 Scala sources to /home/seglo/source/embedded-kafka/embedded-kafka/target/scala-2.12/classes ...
[success] Total time: 5 s, completed 14-Dec-2019 5:15:20 PM
[info] Setting Scala version to 2.11.12 on 3 projects.
[info] Reapplying settings...
[info] Set current project to embedded-kafka-root (in build file:/home/seglo/source/embedded-kafka/)
[success] Total time: 2 s, completed 14-Dec-2019 5:15:22 PM
```

Reply:
Oh weird, I guess my `ammonite` was loading an old 2.12 then. My bad.